tests/test_transformers.py

```diff
@@ -281,3 +283,38 @@ def test_featureunion_returns_concatenated_df(categorical, numerical):
     assert isinstance(transform_df, pd.DataFrame)
     assert 8 == len(transform_df.columns)
     assert len(df) == len(transform_df)
+
+
+def test_dfsimpleimputer_stores_and_returns_correct_values(numerical_na, categorical_na):
+    means = list(np.mean(numerical_na, axis=0))
+
+    imputer_numerical = DFSimpleImputer(strategy='mean')
+    transform_numerical = imputer_numerical.fit_transform(numerical_na)
+
+    numerical_na.iloc[:, 0].fillna(means[0], inplace=True)
+    numerical_na.iloc[:, 1].fillna(means[1], inplace=True)
+
+    imputer_categorical = DFSimpleImputer(strategy='constant', fill_value='missing')
+    transform_categorical = imputer_categorical.fit_transform(categorical_na)
+
+    assert means == list(imputer_numerical.statistics_)
+    assert numerical_na.equals(transform_numerical)
+    assert 2 == (transform_categorical == 'missing').sum().sum()
+
+
+def test_dfsimpleimputer_returns_dataframe(numerical_na):
```

Comment:
👍 for also testing inside a pipeline

Reply:
Would also add a test including a FeatureUnion, since having incorrect indexes can mess up the union